### DISCUSSION TITLE
Crash when opening a project created with beta 6

### DISCUSSION THREAD

**Discussion by Henri75** [apohl79/audiogridder#765] [Help]
Hi,

I used beta 6 yesterday to create a new project (cubase/catalina) and work for a few hours without problem.

I opened it today (so first time I open the project after saving)... and it is crashing cubase when loading the AG insert !

Here are the logs and trace
[AudioGridderTray_2021-10-14_15-16-11.log](https://github.com/apohl79/audiogridder/files/7346290/AudioGridderTray_2021-10-14_15-16-11.log)
[AudioGridderPlugin_2021-10-14_15-16-11.log](https://github.com/apohl79/audiogridder/files/7346291/AudioGridderPlugin_2021-10-14_15-16-11.log)

[AudioGridderPlugin_2021-10-14_15-16-11.trace.zip](https://github.com/apohl79/audiogridder/files/7346295/AudioGridderPlugin_2021-10-14_15-16-11.trace.zip)

**Comment by Henri75** [comment]
I tested again AG in a new project.
After more than one hour of rescanning lots of plugins already scanned, and a lot of enable/disable to stabilize connections, I manage to make it work on a large projects with more than a dozen AG inserts despite the random crashes of AG server while mixing.

Bad news is that when I reopen the project (which was working one hour ago when I saved it), AG crashes and then crash cubase !!!!

Here are some new logs 
[AG CRASH on project opening.zip](https://github.com/apohl79/audiogridder/files/7605144/AG.CRASH.on.project.opening.zip)

**Comment by yoshimodular** [comment]
I have the same issue here with Cubase. When I take AGridder off the folder, the project opens perfectly.

**Reply by yoshimodular** [reply]
BTW, I've tried not to have the server open, so it's something with the client plugin.
